website/docs/r/spring_cloud_config_server.html.markdown

```diff
@@ -0,0 +1,121 @@
+---
+subcategory: "App Platform"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_spring_cloud_config_server"
+description: |-
+  Manages an Azure Spring Cloud Config Server.
+---
+
+# azurerm_spring_cloud_config_server
+
+Manages an Azure Spring Cloud Config Server.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "Southeast Asia"
+}
+
+resource "azurerm_spring_cloud" "example" {
+  name                = "sc-example"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+
+  tags = {
+    environment = "staging"
+  }
+}
+
+resource "azurerm_spring_cloud_config_server" "example" {
+  label           = "config"
+  spring_cloud_id = azurerm_spring_cloud.example.id
+  uri             = "https://github.com/Azure-Samples/piggymetrics"

```

Comment:
this example doesn't match the resource defined above - as such when we integrate it into the resource can we fix this? :)